docs/markdown/Python/python-integrations/protobuf-python.md

```diff
@@ -107,10 +107,24 @@ protobuf_sources(
 )
 ```
 
+> 📘 Use alternative grpc plugins
+>
+> By default, the `grpcio` plugin is used to generate the services stubs. Pants can also generate service stub for [grpclib](https://github.com/vmagamedov/grpclib).
+>
+> To use [grpclib](https://github.com/vmagamedov/grpclib), set `grpclib_plugin = true` in the `[python-protobuf]` scope. You probably also want to disable the `grpcio` plugin if you don't use it.
```

Comment:
Great documentation! Thanks for writing this.

Reply:
Thanks! I don't think I'm that great of a writer, but hopefully this is better than nothing and save other people time ;)